docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+---
+title: "Tanzu Kubernetes Service"
+date: 2020-11-16T13:34:31-05:00
+weight: 9
+description: "Sample for using the operator to set up a WLS cluster on the Tanzu Kubernetes Service."
+---
+
+
+This sample demonstrates how to use the Oracle [WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter “the operator”) to set up a WebLogic Server (WLS) cluster on the Tanzu Kubernetes Grid(TKG).
+After going through the steps, your WLS domain runs on an TKG Kubernetes cluster instance and you can manage your WLS domain by accessing the WebLogic Server Administration Console.
```

Comment:
After going through the steps, your WLS domain  with domain source type of Model In Image runs on an TKG Kubernetes cluster instance.  Once the domain has been provisioned you can monitor it using the WebLogic Administration console.